docs/en/setup/backend/backend-setup.md

```diff
@@ -19,15 +19,18 @@ Requirement: **Java 11/17/21**.
 You should set up the database ready before starting the backend. We recommend to use BanyanDB.
 If you want to use other databases, please read the [storage document](backend-storage.md).
 
-Use the docker mode to run BanyanDB containerized.
+Use the docker mode to run BanyanDB containerized, check 
```

Comment:
Is there anything missed here?

Reply:
Fixed. Nothing needs to say. The tip has been added into comments of shell.